agent/stats/queue.go

```diff
@@ -48,12 +46,13 @@ func NewQueue(maxSize int) *Queue {
 	}
 }
 
-// Reset resets the stats queue.
+// Reset sets all metrics in the queue to 'sent' status
```

Comment:
nit: should this method be renamed? "setting all metrics to sent" doesn't read like it's "resetting".

Reply:
yeah i suppose from the outside it's still a 'reset' though, since it means that from thenceforth only new metrics will be considered valid for sending onwards, so from the perspective of a consumer of the queue, the behavior hasn't changed at all.

Reply:
I changed the comment to reflect what it's doing, what do you think?

Reply:
that comment def helps 👍